modules/importer/src/main/scala/ConfigReader.scala

```diff
@@ -37,26 +37,33 @@ object ConfigReader3 {
     def enum[A: Enumerated: TypeTag](f: A => String): Read[A] =
       cast[String].map(ufindp(f))
 
+    def seqOpt[S <: SequenceableSpType, A: Enumerated: TypeTag](f: A => String, default: S): Read[A] =
+      cast[edu.gemini.shared.util.immutable.Option[S]].map { opt =>
+        val s = if (opt.isEmpty) default else opt.getValue
+        ufindp(f)(s.sequenceValue)
+      }
+
```

Comment:
Turns out that "staff-only" fields are added as instances of Gemini `Option`.  This is some rather gross code to extract a value from the sequence when we know it is one of these types.